maven-java/kalix-maven-archetype-event-sourced-entity/src/main/resources/archetype-resources/docker-compose.yml

```diff
@@ -5,7 +5,7 @@
 version: "3"
 services:
   kalix-proxy:
-    image: gcr.io/kalix-public/kalix-proxy:1.1.22]]#
```

Comment:
what is this?

Reply:
this came from https://github.com/lightbend/kalix-jvm-sdk/pull/1847/files#diff-ae45dca3d7d520f1111eb3e1dd9929349ac20dd90a0fa033ab684918e037ca94R8-R10 

And I guess now the logic we use to open the PR with the bot is not smart enough to let the "]]#" there.. although I wonder what is that doing?  @aludwiko are the "]]#" really required here?

Reply:
yes, it's a part of the velocity template, I need to escape this part to use a variable a line below

Reply:
I assume we cannot do the escaping piece on the line below though?

Something like: `    ]]#container_name: ${artifactId}` ?

Reply:
well, maybe we can, I haven't thought about it